guardrails/src/domain/value_objects/revision.py

```
@@ -0,0 +1,36 @@
+
+@dataclass(frozen=True)
+class Revision:
+    """
+    Monotonic version counter for guardrail configurations.
+    """
+
+    value: int
+
+    def __post_init__(self):
+        if self.value < 1:
+            raise ValueError("Revision must be >= 1")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)